.github/CHANGELOG.md

```diff
@@ -84,6 +84,9 @@
 
 <h3>Improvements</h3>
 
+* The Hamiltonian class was moved to the `ops/qubit` folder, since it is now an observable.
```

Comment:
```suggestion
* The Hamiltonian class was moved to the `ops/qubit` folder from the `vqe` module, since it is now an observable.
```

Could anyone use `qml.vqe.Hamiltonian` and be looking for it's path there?

Reply:
That's a good point! I could try to import the class in `vqe` for the Deprecation cycle so one can import it from there.